docs/source/conf.py

```diff
@@ -16,7 +16,18 @@
 
 import os
 import sys
-import subprocess
+
+
+def getstatusoutput(cmd):
```

Comment:
I don't think we need create new method, moreover, no need redirect stderror to stdoutput

Reply:
I can not confirm if redirection is necessary or not.